x-pack/qa/multi-cluster-search-security/legacy-with-basic-license/src/test/resources/rest-api-spec/test/querying_cluster/80_esql.yml

```diff
@@ -86,35 +86,24 @@ teardown:
         ignore: 404
 
 ---
-"Index data and search on the mixed cluster":
+"ES|QL cross-cluster query fails with basic license":
```

Comment:
Shouldn't we add some non-basic test to replace this one?

Reply:
I tried moving the original test to `x-pack/qa/multi-cluster-search-security/legacy-with-full-license`, but after an hour of trying gave up, as the setup in that other directory is not the same and it will take some time to make it so.

My view is that there is now a lot of testing around this feature. I added quite a few tests around it (and adjusted existing ones). And I like that this basic test is now testing failure when not having the enterprise license. That is they key test to add.

Reply:
And there are other REST tests that have the trial license installed that test ES|QL CCS to ensure it works with that license - those are ones that didn't break in this PR. Example: RemoteClusterSecurityEsqlIT